cmd/libsnap-confine-private/cgroup-support.c

```diff
@@ -90,12 +91,13 @@ bool sc_cgroup_is_v2(void) {
         die("cannot statfs %s", cgroup_dir);
     }
     if (buf.f_type == CGROUP2_SUPER_MAGIC) {
-        if (!did_warn) {
+        if (!did_warn && !hide_warning) {
             fprintf(stderr, "WARNING: cgroup v2 is not fully supported yet, proceeding with partial confinement\n");
             did_warn = true;
         }
         return true;
     }
+    free(hide_warning);
```

Comment:
hide_warning is no longer a pointer to dynamically allocated memory?

Reply:
Ugh... :man_facepalming: of course not. Fixed.